admin_opa_version_integration_test.go

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package tfe
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAdminOpaVersions_List(t *testing.T) {
+	skipUnlessEnterprise(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	t.Run("without list options", func(t *testing.T) {
+		oList, err := client.Admin.OpaVersions.List(ctx, nil)
+		require.NoError(t, err)
+
+		assert.NotEmpty(t, oList.Items)
```

Comment:
Please correct me if I am wrong, but I thought these tool versions are prepopulated in the test instances from this script here: https://github.com/hashicorp/atlas/blob/f016d274ae6bb7d4da54fdeede86a794125dbaac/lib/data/tool_versions.yml#L1028
Because it looks like [admin_terraform _versions tests ](https://github.com/hashicorp/go-tfe/blob/main/admin_terraform_version_integration_test.go#L15)do the same thing.